src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -47,11 +47,20 @@ const AuctionCollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
       return null
     }
 
+    const lotClosesAt =
+      !activeLotData.sale.live_start_at &&
+      (activeLotData.saleArtwork.extended_bidding_end_at ||
+        activeLotData.saleArtwork.end_at ||
+        activeLotData.sale.ended_at)
+    const isBiddingClosed = lotClosesAt && new Date(lotClosesAt) <= new Date()
+
     // Resolve all associated auctions data in one object
     return {
       artwork,
       saleArtwork: activeLotData.saleArtwork,
       sale: activeLotData.sale,
+      lotClosesAt,
+      isBiddingClosed,
```

Comment:
Rather than loading these values up here, I think the preferred pattern is for this to go in the resolver below. We can use `sale` and `saleArtwork` down there, included because this parent resolver loads them here.

However,
- if the `isBiddingClosed` property means that the entire `auction` field could be null, we could do that here
- if we want to re-use the `lotClosesAt` logic down below, we can extract a function that takes the same sale/saleArtwork args for both.